backend/api/v1/endpoints/connections.py

```
@@ -56,13 +56,38 @@ def list_connections(db: Session = Depends(get_db), current_user: User = Depends
+        friends = db.query(Connection.friend_id).filter(Connection.user_id == current_user.id)
+
+        # Fetch users who have been sent a request by the current user
+        sent_requests = db.query(Connection.friend_id).filter(
+            Connection.user_id == current_user.id, 
+            Connection.status == "pending"
+        )
+
+        # Fetch users who have sent a request to the current user
+        received_requests = db.query(Connection.user_id).filter(
+            Connection.friend_id == current_user.id, 
+            Connection.status == "pending"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)